templates/repo/settings/lfs_file.tmpl

```diff
@@ -0,0 +1,57 @@
+{{template "base/head" .}}
+<div class="repository settings lfs">
+	{{template "repo/header" .}}
+	{{template "repo/settings/navbar" .}}
+	<div class="ui container repository file list">
+		{{template "base/alert" .}}
+		<div class="tab-size-8 non-diff-file-content">
+			<h4 class="ui top attached header">
+				<a href="{{.LFSFilesLink}}">{{.i18n.Tr "repo.settings.lfs"}}</a> / <span class="truncate sha">{{.LFSFile.Oid}}</span>
+				<div class="ui right">
+					<a class="ui blue show-panel button" href="{{.LFSFilesLink}}/find?oid={{.LFSFile.Oid}}&size={{.LFSFile.Size}}">{{$.i18n.Tr "repo.settings.lfs_findcommits"}}</a>
+				</div>
+			</h4>
+			<div class="ui attached table unstackable segment">
+				<div class="file-view {{if .IsMarkup}}markdown{{else if .IsRenderedHTML}}plain-text{{else if .IsTextFile}}code-view{{end}} has-emoji">
+					{{if .IsMarkup}}
+						{{if .FileContent}}{{.FileContent | Safe}}{{end}}
+					{{else if .IsRenderedHTML}}
+						<pre>{{if .FileContent}}{{.FileContent | Str2html}}{{end}}</pre>
+					{{else if not .IsTextFile}}
+						<div class="view-raw ui center">
+							{{if .IsImageFile}}
+								<img src="{{EscapePound $.RawFileLink}}">
+							{{else if .IsVideoFile}}
+								<video controls src="{{EscapePound $.RawFileLink}}">
+									<strong>{{.i18n.Tr "repo.video_not_supported_in_browser"}}</strong>
+								</video>
+							{{else if .IsAudioFile}}
+								<audio controls src="{{EscapePound $.RawFileLink}}">
+									<strong>{{.i18n.Tr "repo.audio_not_supported_in_browser"}}</strong>
+								</audio>
+							{{else if .IsPDFFile}}
```

Comment:
Should we support custom renderers? Perhaps as a future PR?

Reply:
? is this a new feature that I've missed.

Reply:
I've seen that custom renderers were affected by a regression and then a PR fixed that. May be it doesn't apply here? #4105 and #7869

Reply:
Ah we don't know the name so we can't actually attempt to render with markup! I knew there was a reason why I hadn't implemented this.